shell/server/core.js

```diff
@@ -205,13 +247,20 @@ SandstormCoreImpl.prototype.drop = function (sturdyRef) {
 SandstormCoreImpl.prototype.makeToken = function (ref, owner) {
   var self = this;
   return inMeteor(function () {
-    var sturdyRef = new Buffer(Random.id(20));
+    if (owner.grain && !owner.grain.introducerUser) {
+      // In cases where the caller doesn't know the introducerUser to save under (eg. in the case of
```

Comment:
Actually after just discussing with Kenton, we do want to allow `introducerUser` to be empty. In the case where it is empty, `requiredPermissions` will be ignored.